Documentation/ApiOverview/SiteHandling/SiteSets.rst

```diff
@@ -0,0 +1,345 @@
+..  include:: /Includes.rst.txt
+..  index:: Site handling; Site sets
+..  _site-sets:
+
+=========
+Site sets
+=========
+
+Site sets ship parts of the site configuration as composable pieces. They are
+intended to deliver settings, TypoScript, TSconfig and reference enabled content
```

Comment:
Maybe cross-reference links to TypoScript and TSConfig?

Is the "content block" reference already available, or only for future TYPO3 versions?

Reply:
woops, thanks for catching this! I think it is a future reference as the content blocks are not yet merged to my knowledge

Reply:
Done